backend/baseclass.pm

```diff
@@ -1349,6 +1349,13 @@ sub stop_ssh_serial {
     return $self->{serial} = undef;
 }
 
+sub hide_password
+{
```

Comment:
For consistency, move the `{` to the line above.

Reply:
sure thing, thx. Funny that tiny doesn't complain.